plugins/MauticFocusBundle/Resources/views/Builder/form.html.twig

```diff
@@ -45,21 +45,23 @@
             document.getElementById('mauticform{{ formName }}_' + messageType).innerHTML = message;
         }
 
-        setTimeout(function () {
-            if (headline.length) {
-                {% if 'bar' == style %}
-                headline[0].style.display = "inline-block";
-                {% else %}
-                headline[0].style.display = "block";
-                {% endif %}
-            }
-            if (tagline.length) {
-                tagline[0].style.display = "inherit";
-            }
+        if (messageType == 'error') {
+            setTimeout(function () {
+                if (headline.length) {
+                    {% if 'bar' == style %}
+                    headline[0].style.display = "inline-block";
```

Comment:
The question is that this isn't going to be set for other than `error` messages. I'm not sure if it matters.

Reply:
Focus form on success/error hide form and display error/success message and then back again to form.
Now it will do it just for error. Success message keep on popup/window

Seems OK to me.